app/2.0/docs/devguide/style-shadow-dom.md

```diff
@@ -515,6 +515,72 @@ class myElement extends Polymer.Element(){
 }
 ```
 
+### Style directional text with the :dir() selector
+
+The `:dir()` CSS selector allows for styling text specific to its orientation 
+(right-to-left or left-to-right). See the [documentation on MDN](https://developer.mozilla.org/en-US/docs/Web/CSS/:dir) for more information on the `:dir()`
+selector.
+
+The `Polymer.DirMixin` provides limited support for the `:dir()` selector. Use of `:dir()` requires the
+application to set the `dir` attribute on `<html>`. All elements will use the same direction.
+
+Individual elements can opt-out of the global direction by setting the `dir` attribute
+in HTML or at `ready()`, but the text direction of these elements must from then on be handled 
```

Comment:
at `ready()` => in the `ready` callback. (In docs, leave off the parens for methods unless you're including arguments or distinguishing different versions of a method/function).